lib/tool_belt/cherry_picker.rb

```diff
@@ -27,7 +27,7 @@ def find_cherry_picks(project, release, repo_names)
         commits = issue['changesets']
 
         commits.each do |commit|
-          if !commit['comments'].start_with?('Merge pull request') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
+          if commit['comments'].downcase.start_with?('fixes', 'refs') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
```

Comment:
Those should be the only legitimate prefixes to a commit that should be cherry-picked. Right? I thought that was the agreed upon standard; refs if it's still in progress and fixes if it's complete.